swagger.yaml

```diff
@@ -155,5 +317,116 @@ components:
           format: date
         assignedPersons:
           type: array
-          items: 
-            type: integer
\ No newline at end of file
+          items:
+            type: integer
+
+    Leads:
+      type: object
+      description: Lead data from Pipedrive API
+      properties:
+        id:
+          type: string
+        title:
+          type: string
+        interested:
+          type: string
+          description: "Might need to be changed to Pipedrive API item-key"
+        usedTech:
+          type: string
+          description: "Might need to be changed to Pipedrive API item-key"
+        add_time:
```

Comment:
Even if pipedrive is using snake_case you could map over the response to rename for our use using camelCase- which would be very nice, and these all these snake_case properties can be renamed